src/start/RealmInputScreen.js

```diff
@@ -126,12 +213,52 @@ export default function RealmInputScreen(props: Props): Node {
           fontSize: 20,
           color: themeContext.color,
         },
-        hintText: { paddingLeft: 2, fontSize: 12 },
+        suggestionText: { fontSize: 12, fontStyle: 'italic' },
+        suggestionTextLink: {
+          fontStyle: 'normal',
+          color: BRAND_COLOR, // chosen to mimic WebLink
+        },
         button: { marginTop: 8 },
       }),
     [themeContext],
   );
 
+  const suggestionContent = React.useMemo(() => {
+    if (suggestion === null) {
+      // Vertical spacer so the layout doesn't jump when a suggestion
+      // appears or disappears. (The empty string might be neater, but it
+      // doesn't give the right height… probably lots of people wanted it to
```

Comment:
Sure, `' '` seems fine.

Another variation to try, if it seems cleaner, would be `'\u200b'`, i.e. U+200B ZERO WIDTH SPACE.